pennylane/optimize/adagrad.py

```diff
@@ -40,9 +40,9 @@ class AdagradOptimizer(GradientDescentOptimizer):
 
     :math:`\eta` is the step size, a user defined parameter.
 
-    Args:
-        stepsize (float): the user-defined hyperparameter :math:`\eta`
-        eps (float): offset :math:`\epsilon` added for numerical stability
+    Parameters:
```

Comment:
Very _very_ minor flyby comment, but while `Parameters` and `Args` are aliases in Sphinx autodoc, I think by convention we use `Args:` everywhere else in docstrings in PL.

(Note: no matter which one you choose, it always renders as **Parameters** on the website!)